_posts/0000-01-02-RDogPinK.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Really? I should change code within the pull request? Is this updating my branch too or does it lead to a mess?!